admin/osx/make_universal.py

```diff
@@ -0,0 +1,81 @@
+#!/usr/bin/env python
+
+import sys
+import os
+import subprocess
+
+
+# A general note: We first produce a x86_64 and a arm64 app package
+# and then merge them together instead of compiling the desktop client
+# with the CMake option CMAKE_OSX_ARCHITECTURES="x86_64;arm64" because
+# macdeployqt can not handle universal binaries well. In the future
+# with Qt6 this might change and this script will become obsolete.
+
+
+def usage(program_name):
+    print("Creates a universal app package from a x86_64 and a arm64 app package.")
+    print("Usage: {} x86_64_app_file arm64_app_file output_directory".format(program_name))
```

Comment:
what is the app_file?

Reply:
The directory ending with `.app` for example `Nextcloud.app`. I named it file because for the user it just looks like one file. We could also name it app package.

Reply:
Okay, I added an example to make it more clear.